README.md

```diff
@@ -94,6 +94,9 @@ Here is listed the basic API of both KtdGridComponent and KtdGridItemComponent.
 /** Layout of the grid. Array of all the grid items with its 'id' and position on the grid. */
 @Input() layout: KtdGridLayout;
 
+/** Grid gap in css pixels */
+@Input() gridGap: number = 0;
```

Comment:
What about renaming 'gridGap' for just 'gap'? We are on a 'grid' context, so 'grid' prefix is not necessary.